default.hbs

```diff
@@ -34,6 +34,8 @@
         <div class="site-content">
             {{{body}}}
         </div>
+        {{> our-partners}}
+        {{> featured-by}}
```

Comment:
These two should be placed just on the homepage (`home.hbs`), not in every page.